src/components/filters/FilterButtonsContainer.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import { Button } from '../ui/button';
+import { PiSlidersHorizontalBold } from 'react-icons/pi';
+import { LuArrowDownUp } from 'react-icons/lu';
+
+const FilterButtonsContainer: React.FC = () => {
+  const handleFilterClick = () => {
+    alert('Filter button clicked');
+    // logic to open the filter modal here or modalContext
+  };
+
+  const handleSortClick = () => {
+    alert('Sort button clicked');
+    // logic to open the sort action sheet or dropdown here, or modalContext
+  };
+
+  return (
+    <div className='mb-6 grid grid-cols-1 gap-4 sm:grid-cols-2 md:flex md:justify-start lg:hidden'>
+      <Button
+        variant='filter'
+        icon={<PiSlidersHorizontalBold />}
+        iconPosition='left'
+        className='px-5'
+        onClick={handleFilterClick}
+      >
+        Filter by
+      </Button>
+      <Button
+        variant='filter'
+        icon={<LuArrowDownUp />}
+        iconPosition='left'
+        className='px-5'
```

Comment:
🟡 since both filter buttons have the same class of `px-5`, this could be a part of the `'filter'` button variant, to improve UI consistency and DRY :)

Reply:
Sure, can change this. Thanks!